css/main.css

```diff
@@ -0,0 +1,265 @@
+/* Style based on the NEDERBURG Hugo theme (https://themes.gohugo.io/hugo-nederburg-theme/) */
```

Comment:
The licence (https://github.com/appernetic/hugo-nederburg-theme/blob/master/LICENSE) requires that the copyright statement and licence text in included in all copies of substantial portions of the software. A link is not sufficient.

Reply:
> included in all copies of substantial portions of the software

I have added a copy of the nederburg license. I should note that I implemented this file from scratch by inspecting the demo website (I can't really read complex CSS files). I think it still qualifies as derivative work though.